### ISSUE TITLE
getFollowedTimelineSinceID issue

### ISSUE THREAD

**Issue by sweetie85** [mattgemmell/MGTwitterEngine#64]
Hi,

I have  a problem with getFollowedTimelineSinceID method. I am using this method to get tweets. 
But I am getting only 20 tweets, though I am changing the count but still its pulling only 20 tweets.

Can you please guide me on how to pull all the messages from timeline.

Thanks,
Swetha.

**Comment by mginod** [comment]
I have the exact same problem. It seems the 'page' and 'count' parameters have no effect in the getFollowedTimelineSinceID method. There are always 20 results.

**Comment by sweetie85** [comment]
I researched for this problem, but could not find any solution, can you please let me know if you find any solution.

Thanks.

**Comment by arpandesai** [comment]
Hello All, 

Typical Problem I faced for a week or so.

You just need to remove the 'Comment Lines'

Problem is that your parameters are you not appending to the request Twitter Engine is sending.

  SA_OAuthTwitterEngine.m > _sendRequestWithMethod (Method)

```
// modificaiton from the base clase
// the base class appends parameters here
```

Remove the Comments on this code :

 if (params) {
            fullPath = [self _queryStringWithBase:fullPath parameters:params prefixed:YES];
       }

Regards,
Arpan

**Comment by FreeApple** [comment]
This works for me, but then blocks me from posting. If I remove the "comment lines" then I can call out the "count" and it seems to work "which I might add I have tried forever to get more then 20 tweets!"

Then now when I try to post a tweet since I am using xAuth I get a 401 error saying my user name and password cannot be verified....but I can do anything else and I am logged in as you can clearly see y username and all my tweets so there is a bug somewhere in the parameters! 

If anyone has any more insight on this topic please enlighten me

**Comment by SepiaApps** [comment]
I'm running into the same exact scenario as FreeApple. Anyone have any solutions?

**Comment by FreeApple** [comment]
Yeah that was a hard one to figure out. I should have came back with my solution earlier. So after I unblocked this code I used if statements such as 

```
if (requestType == MGTwitterStatus) {
    NSLog(@"Statuses Requested");

   if (params) {
        fullPath = [self _queryStringWithBase:fullPath parameters:params prefixed:YES];
   }

}
if (requestType == MGTwitterRepliesRequest) {
    if (params) {
        fullPath = [self _queryStringWithBase:fullPath parameters:params prefixed:YES];
    }
}
if (requestType == MGTwitterDirectMessagesRequest) {
    if (params) {
        fullPath = [self _queryStringWithBase:fullPath parameters:params prefixed:YES];
    }

}

if (requestType == MGTwitterUserTimelineForUserRequest) {
    if (params) {
        fullPath = [self _queryStringWithBase:fullPath parameters:params prefixed:YES];
    }
}            
```

Hope this helps others !
